src/app/app.component.ts

```diff
@@ -1,8 +1,26 @@
 import { Component } from "@angular/core";
+import { DialogType } from "projects/core/src/lib/models/dialogType";
+import { DialogService } from "projects/core/src/lib/services/dialog.service";
 
 @Component({
 	selector: "jsmu-root",
 	templateUrl: "./app.component.html",
 	styleUrls: ["./app.component.scss"]
 })
-export class AppComponent {}
+export class AppComponent {
+	public dialogType!: DialogType;
+	public isOpen!: boolean;
+
+	constructor(private dialogService: DialogService) {
+		dialogService.$dialogType.subscribe((dialogType: DialogType) => {
+			this.dialogType = dialogType;
+		});
+		dialogService.$isOpen.subscribe((isOpen: boolean) => {
+			this.isOpen = isOpen;
+		});
+	}
```

Comment:
instead of using a subscription, it is better to use the async pipe for more conciseness and convenience

Reply:
is was, but in this case it is necessary to use dialog service in template, you said it is wrong. is there any other way for using async pipe?

Reply:
in class:
this.isOpenDialod$ = this.dialogService.$isOpen;

in template:
isOpenDialod$ | async

Reply:
done